File: genium/src/test/resources/smoke/inheritance/output/android/com/example/smoke/ChildClassFromClass.java
/*
 *

 */
package com.example.smoke;
public final class ChildClassFromClass extends ParentClass {
    /**
     * For internal use only.
     * @exclude
     */
    protected ChildClassFromClass(final long nativeHandle) {
        super(nativeHandle);
    }
    public native void childClassMethod();
}

File: gluecodium/src/test/resources/smoke/inheritance/output/android-kotlin/com/example/smoke/ChildClassFromClass.kt
/*

 *
 */

@file:JvmName("ChildClassFromClass")

package com.example.smoke


class ChildClassFromClass : ParentClass {



    /**
     * For internal use only.
     * @suppress
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, tag) {}





    external fun childClassMethod() : Unit




}

